packages/mongodb/src/db/initDb.ts

```diff
@@ -1,27 +1,21 @@
 import { MongoInternals } from 'meteor/mongo';
-import { Db } from 'mongodb';
+import { Db /* MongoClient */ } from 'mongodb';
 
-// import { MongoClient } from 'mongodb';
-// import { log, LogTextColor } from './utils/log';
+// const url = process.env.MONGO_URL || 'mongodb://localhost:4011';
 
-// or as an es module:
-// import { MongoClient } from 'mongodb'
+// const initDbNative = async (): Promise<Db> => {
+//   const client = new MongoClient(url);
 
-// Connection URL
-// const url = process.env.MONGO_URL || 'mongodb://localhost:27017';
-// const client = new MongoClient(url);
+//   const dbName = 'meteor';
+//   await client.connect();
+//   const db = client.db(dbName);
+//   return db;
+// };
 
-const initDb = (): Db => {
-  /* TODO
-  // Use connect method to connect to the server
-  log('Initialise MongoDB database', { color: LogTextColor.Crimson });
-  await client.connect();
-  log('Client successfully connected', { color: LogTextColor.Crimson });
-  const db = client.db(dbName);
-  log('Dabase successfully initialised', { color: LogTextColor.Crimson });
-  return db
-  */
+const initDbMeteor = async (): Promise<Db> => {
   return MongoInternals.defaultRemoteCollectionDriver().mongo.db as unknown as Db;
```

Comment:
Only a small detail, but the `as unknown as Db` seems superfluous to me.
`.mongo.db` is of type `MongoDb` and there should not be a type error without the hints